.evergreen/csfle/mock_kms.js

```diff
@@ -1,5 +1,18 @@
 /**
- * Start a KMS mock server using variables from kms_setup.json
+ * Start a KMS mock server using variables from kms_setup.json.
+ *
+ * To run the KMS mock server from this directory on port 8000 with a particular CA and cert file,
+ * use the following bash snippet.
+ *
+ *   cat <<EOF > kms_setup.json
+ *   {
+ *      "kms_ca_file": "path/to/ca_file",
+ *      "kms_cert_file": "path/to/cert_file",
+ *      "port": "8000"
+ *   }
+ *   EOF
+ *   mongo --nodb mock_kms.js
```

Comment:
What value does the `mock_kms.js` script provide? It seems like it's just a trivial wrapper for `kms_http_server.py`.

Reply:
Yeah... it's actually more complicated to use `mock_kms.js`; I just copied it from the server's code. Removed it in favor of `kms_http_server.py` and added a usage example.